src/ddl/fluview.sql

```diff
@@ -345,25 +345,40 @@ Note that the flusurv age groups are, in general, not the same as the ILINet
 particular "catchment" (e.g. 'network_all', 'CA', 'NY_albany') rather than by
 regions and states in general.
 
-+--------------+-------------+------+-----+---------+----------------+
-| Field        | Type        | Null | Key | Default | Extra          |
-+--------------+-------------+------+-----+---------+----------------+
-| id           | int(11)     | NO   | PRI | NULL    | auto_increment |
-| release_date | date        | NO   | MUL | NULL    |                |
-| issue        | int(11)     | NO   | MUL | NULL    |                |
-| epiweek      | int(11)     | NO   | MUL | NULL    |                |
-| location     | varchar(32) | NO   | MUL | NULL    |                |
-| lag          | int(11)     | NO   | MUL | NULL    |                |
-| rate_age_0   | double      | YES  |     | NULL    |                |
-| rate_age_1   | double      | YES  |     | NULL    |                |
-| rate_age_2   | double      | YES  |     | NULL    |                |
-| rate_age_3   | double      | YES  |     | NULL    |                |
-| rate_age_4   | double      | YES  |     | NULL    |                |
-| rate_overall | double      | YES  |     | NULL    |                |
-| rate_age_5   | double      | YES  |     | NULL    |                |
-| rate_age_6   | double      | YES  |     | NULL    |                |
-| rate_age_7   | double      | YES  |     | NULL    |                |
-+--------------+-------------+------+-----+---------+----------------+
++-------------------+-------------+------+-----+---------+----------------+
+| Field             | Type        | Null | Key | Default | Extra          |
++-------------------+-------------+------+-----+---------+----------------+
```

Comment:
i feel like these textual "tables" scattered in the comments of many files in this directory are at least not necessary and at most a waste of space...  they dont have any more information than the SQL table definitions below, and theyre a pain to keep in sync.  id argue for removing them, but since you already went to the trouble of updating it properly, we can keep it.

Reply:
I agree about being a pain to keep up to date! I guess I'll leave this so it matches the format of the other tables defined in this file, and we can remove in a separate PR if we want.